tests/Contents.m

```diff
@@ -0,0 +1,40 @@
+% TESTS Unit tests for Rigbox.
```

Comment:
I would change this to 'unit and integration tests'. I think the line between unit and integration tests is ill-defined, but I typically think of a unit test as a test for a single function or method, while a lot of the Rigbox test functions test multiple functions within a single test